contentstore/client.go

```diff
@@ -48,6 +48,7 @@ func (p *ContentStoreClient) DoRequest(httpMethod, path string, data []byte) (*h
 
 	if data != nil {
 		req.Header.Set("Content-Type", "application/json")
+		req.Header.Set("Accept", "application/json")
```

Comment:
This wants to be outside the conditional.  We want to set an `Accept` header even if we're not sending a request body (eg for a GET request).